x-pack/plugins/maps/public/routing/routes/maps_app/maps_app_view.js

```diff
@@ -102,14 +102,6 @@ export class MapsAppView extends React.Component {
       this._globalSyncChangeMonitorSubscription.unsubscribe();
     }
 
-    // Clean up app state filters
```

Comment:
No longer need to clean-up on unmount since app state filters gets cleaned up on mount